packages/stories/src/index.tsx

```diff
@@ -40,10 +42,29 @@ export default function Stories({
     }
   }, [onStoriesStart]);
 
+  const preloadStory = (
```

Comment:
can we move this function to utilities function. 
also lets keep the signature simple  example 
```
function preloadStory(type: image|video, url:string){
  if(type === 'image'){
     preloadImage(url);
     return
  }
  preloadVideo(url)
}
```